app/models/manageiq/providers/ibm_cloud/inventory/collector/object_storage.rb

```diff
@@ -11,7 +11,7 @@ def buckets
     (connection.list_buckets[:buckets] || []).each do |bucket|
       connection.get_bucket_location(:bucket => bucket[:name]).location_constraint
       buckets << bucket
-    rescue
+    rescue Aws::S3::Errors::NoSuchBucket
```

Comment:
NOTE this overly broad rescue was hiding the `VCR::Errors::UnhandledHTTPRequestError` exception